### PR TITLE
88.implement custom vibe selector

### LINKED ISSUE #88: Implement Custom Spinner or Selector for Picking a Vibe
Description:
As of now, currently have a spinner to select a Vibe but we want to implement a custom selector similar to that of a Date or Time picker. We have the option of making a custom spinner or selector.

For a custom spinner, we would need to just add images to the already existing spinner and refactor to handle/use a custom spinner in place of the one we have now. Refer to [1](https://www.codingdemos.com/android-custom-spinner-images-text/) or [2](https://medium.com/@hannaholukoye/adding-an-icon-on-a-spinner-on-android-e99c7bc6c180) for examples/tutorials.

For a custom selector, we could refer to [1](https://stackoverflow.com/questions/41501600/how-to-implement-the-avatar-selection-screen-like-this-in-android-without-using) as an example, or [2](https://medium.com/@nbtk123/create-your-own-horizontal-vertical-slider-picker-android-94b6ee32b3ff). Using the [CarouselPicker](https://github.com/GoodieBag/CarouselPicker) library is another option.

Story Points: 8

Task List
- [x] Decide what kind of selector/spinner/picker we want to implement and tell team.
- [x] Implement chosen picker and make sure it works + integrate our set of emoticons

### PR DESCRIPTION
**Pull request by raysarinas**
[//]: # (These are comments and are used for reference and will not show up in the PR)

Description: Made custom vibe selector instead of dropdown. Changed the vector assets for Sad and Scared emoticons Refer to Issue #88 

[//]: # (Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change)

[//]: # (automatically closes given issue number)

Proof of Concept
Screenshot_1573772663
Screenshot_1573772658
Screenshot_1573772649

[//]: # (Attach Image of feature added or fix applied and working)
[//]: # (Please provide images of tests passing)

Checklist

[//]: # (Add an x between square brackets to mark item as done)

- [x] Follows style guidelines and is **properly commented**
- [x] Code changes generate no new warnings
- [x] Documentation (UML, Wiki, etc.)

### CODE REVIEW

**Comment by raysarinas** [comment]
This issue can be closed once @TheBroMoe has completed the UI/Intent testing for vibe selection

File: app/src/main/java/com/example/vybe/VibeFactory.java [hunk 1]
```diff
@@ -49,4 +49,28 @@ else if (selectedVibe.equals("surprised")) {
 
         return null;
     }
```
Comment (gatilogo) [inline app/src/main/java/com/example/vybe/VibeFactory.java]: Reminder to remove getVibe(string selectedVibe) once we shifted all usages to getVibe(int selectedVibe)

File: app/src/main/res/layout/activity_add_edit_vibe.xml [hunk 2]
```diff
@@ -36,11 +36,19 @@
             android:orientation="vertical"
             android:padding="16dp" >
 
-            <Spinner
-                android:id="@+id/vibe_dropdown"
+            <ImageView
+                android:id="@+id/vibe_selector"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:src="@drawable/ic_no_vibe" />  <!-- Placeholder till we get emoticons to work-->
```
Comment (gatilogo) [inline app/src/main/res/layout/activity_add_edit_vibe.xml]: Remove comment?

Comment (raysarinas) [reply app/src/main/res/layout/activity_add_edit_vibe.xml]: Yes

**Comment by raysarinas** [comment]
Screenshot_1573791063
Screenshot_1573790550

Some changes I made also included dynamically changing colors to reflect a selected Vibe 😌

File: app/src/main/java/com/example/vybe/AddEditVibeActivity.java [hunk 3]
```diff
@@ -101,6 +108,9 @@ protected void onCreate(Bundle savedInstanceState) {
             //TODO: set vibe and socsit dropdrowns
```
Comment (gatilogo) [inline app/src/main/java/com/example/vybe/AddEditVibeActivity.java]: update comment i guess but can do under refactoring once socsit dropdown is finalized

**Pull request merged**